site/content/overview/gateway-api-compatibility.md

```diff
@@ -167,7 +167,8 @@ NGINX Gateway Fabric supports a single GatewayClass resource configured with the
       - `type`: Supported.
       - `requestRedirect`: Supported except for the experimental `path` field. If multiple filters are configured, NGINX Gateway Fabric will choose the first and ignore the rest.
       - `requestHeaderModifier`: Supported. If multiple filters are configured, NGINX Gateway Fabric will choose the first and ignore the rest.
-      - `responseHeaderModifier`, `requestMirror`, `urlRewrite`, `extensionRef`: Not supported.
+      - `urlRewrite`: Supported. If multiple filters are configured, NGINX Gateway Fabric will choose the first and ignore the rest.
```

Comment:
Let's add a note here and in the `requestRedirect` section about the incompatibility between the two filters.